src/components/Plans/PlanPaymentDialog.js

```diff
@@ -145,6 +147,7 @@ const PlanPaymentDialog = ({
   }
 
   function showPayment () {
+    upgradeButtonClick()
```

Comment:
Provide a property `onOpen` and pass the `onOpen={upgradeButtonClick}` to upgrade button component in `src/components/TabLimitModal/index.js`

Reply:
Done.